java/runtime/pom.xml

```diff
@@ -21,53 +22,60 @@
       <artifactId>ray-api</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
-      <groupId>com.typesafe</groupId>
-      <artifactId>config</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>de.ruedigermoeller</groupId>
-      <artifactId>fst</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.github.davidmoten</groupId>
-      <artifactId>flatbuffers-java</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>redis.clients</groupId>
-      <artifactId>jedis</artifactId>
-    </dependency>
     <dependency>
       <groupId>org.apache.arrow</groupId>
```

Comment:
These `dependency` items should have the same indentation as those you changed.

Reply:
The below ones are not indented, because the tool doesn't support that.